index.html

```diff
@@ -1831,322 +1831,291 @@ <h2>Proof</h2>
-  it is NOT RECOMMENDED to define a new context unless the context is necessary to
-  properly implement the method. Method-specific contexts MUST NOT override the
-  terms defined in the generic <a>DID</a> context.
+[[!JSON-LD]] is a JSON-based format used to serialize
+<a href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>.
         </p>
-      </section>
-
-      <section>
-        <h2>JSON-LD Extensibility</h2>
 
         <p>
-One of the goals of the <a>decentralized identifiers</a> data model is to
-enable permissionless innovation. The data model needs to be extensible in a
-number of different ways, including:
+When producing and consuming DID documents that are in JSON-LD (as noted by
+the document metadata), the following rules MUST be followed. 
         </p>
 
         <ul>
-          <li>
-Modelling of complex multi-entity relationships is provided through the use of a
-graph-based data model.
-          </li>
-
-          <li>
-Extending the machine-readable vocabularies used to describe information in the
-data model, without relying on a centralized system, is accomplished through
-the use of [[LINKED-DATA]].
-          </li>
-
-          <li>
-Support for multiple types of cryptographic proof formats is accomplished
-through the use of Linked Data Proofs [[LD-PROOFS]], Linked Data Signatures
-[[LD-SIGNATURES]], and a variety of signature suites.
-          </li>
-
-          <li>
-All of the extensibility mechanisms outlined above are provided in a data format
-that is popular among software developers and web page authors and is enabled
-through the use of [[!JSON-LD]].
-          </li>
+            <li>
+The <code>@id</code> and <code>@type</code> keywords are aliased to
+<code>id</code> and <code>type</code> respectively, enabling developers to use
+this specification as idiomatic JSON.
+            </li>
+            <li>
+Data types, such as integers, dates, units of measure, and URLs, are
+automatically typed to provide stronger type guarantees for use cases that
+require them.
+            </li>
         </ul>
 
-        <p>
-This approach to data modeling is often called an <em>open worldassumption</em>,
-meaning that anyone can say anything about any other thing. This approach often
-seems in conflict with building simple and predictable software systems.
-Balancing extensibility with program correctness is always more challenging with
-an open world assumption than it is with closed software systems.
-        </p>
+        <section>
+            <h2>
+Production
+            </h2>
+
+            <p>
+The <a>DID document</a> is serialized following the rules in the JSON processor, with
+one addition: <a>DID documents</a> MUST include the <code>@context</code> property.
+            </p>
+
+            <dl>
+                <dt><dfn>@context</dfn></dt>
+                <dd>
+The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
+where the value of the one <a>URI</a> is
```

Comment:
```suggestion
where the value of the first <a>URI</a> is
```

Reply:
I'll change this, but it was my understanding that order was not important, so this was meant to be "the value of one URI" (the "the" is extraneous in my edit).

Reply:
Order is vitally important... we're using the JSON-LD 1.1 "@protected" feature, which disallows a term that's defined from being redefined (so people don't accidentally stomp on the base DID context). That means that the base DID context goes first. It also allows JSON-only processors to just check a known order for the `@context` property... but that's no longer a thing, so is less important (maybe).

In any case, order matters, the DID Context must be the first value in the ordered set.

Reply:
To be completely clear: this has no effect on JSON consumers (who ignore the property) or JSON producers (who don't output the property), so it's entirely up to the JSONLD definition, in which order is semantically important.